File: javasec/p001.java
package javasec;

public final class p001 {
	public static void main(String[] args) {
		System.out.println(new p001().calculate());
	}
	public int calculate() {
		int total = 0;
		for (int i = 0; i < 1000; i++) {
			if (i % 3 == 0 || i % 5 == 0) {
				total += i;
			}
		}
		return total;
	}
}

File: csharp/p001.cs
using System;
	
namespace csharp {
	class p001 {
		public int run() {
			int total = 0;
			for (int i = 0; i < 1000; i++) {
				if (i % 5 == 0 || i % 3 == 0) {
					total += i;
				}
			}
			return total;
		}
		static void Main(string[] args) {
			Console.WriteLine(new p001().run());	
		}
	}
}